03_letters/sirigithub/main.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+// letters returns a map of rune literals and its frequency in the given input string s
+func letters(s string) map[rune]int {
+	charecterFreq := make(map[rune]int)
+	// increment the counter if the charecter exists
```

Comment:
I'd suggest removing this comment, as it doesn't say any more than the code already says.

Reply:
That was an oversight, thanks Cam . Done

Reply:
done.

Reply:
oops again yes. Definitely fixed this time.

Reply:
oops again . Definitely fixed this time.